### ISSUE TITLE
Color picker & hover detail make editing clocks difficult/impossible

### ISSUE THREAD

**Issue by 4m2c4bl3** [BluSunrize/fvtt-clockworks#6]
Using Foundry v12 328

The hover text makes editing fields very frustrating, as it pops up directly on top of them.
![image](https://github.com/user-attachments/assets/86d4a83a-d404-4b69-b70f-a19f6810f335)

The color picker means I can't actually paste in hex codes to get the exact colors I want, as it blocks access to the text field.
![image](https://github.com/user-attachments/assets/cff170f2-2d07-44ea-902e-4a32b4316fe2)

**Comment by BluSunrize** [comment]
I agree wit h the hover over text - though I don't believe I can do much about that, foundry just puts that text there and *not* having explaining text kinda sucks too.

I can't reproduce the latter issue though:
![Image](https://github.com/user-attachments/assets/c018d6b2-6b8b-4a5c-a061-e90d0f463e65)

My pickers appear below the field - maybe color picker has updated since you opened this issue and remedied the problem?
